src/avalanchemq/vhost.cr

```diff
@@ -40,8 +40,7 @@ module AvalancheMQ
     @connections = Array(Client).new(512)
     @segments : Hash(UInt32, MFile)
     EXCHANGE_TYPES = %w(direct fanout topic headers
-      x-federation-upstream x-delayed-message
```

Comment:
This makes it incompatible with rabbitmq, no?

Reply:
No, that list only controls which types are listen in the UI. The API looks the same. 

In the UI you instead select which exchange you want and check "Delayed"  checkbox

Reply:
shouldn't `EXCHANGE_TYPES` then be in the http controller if it only affects the UI?